set-operations/01/01_union-all.md

```diff
@@ -0,0 +1,83 @@
+# 01 Union All
+
+:bulb: This example will show how you can use the set operation `UNION ALL` to combine several streams of data.
+
+See [our documentation](https://ci.apache.org/projects/flink/flink-docs-master/docs/dev/table/sql/queries/#set-operations)
+for a full list of fantastic set operations Apache Flink supports.
+
+
+## The Sources
+
+The examples assumes you are building an application that is tracking visits :raccoon: on foreign planets :chestnut:. 
+There are three sources of visits. The univers of Rick and Morty, the very real world of NASA and such, 
+and the not so real world of Hitchhikers Guide To The Galaxy.
+
+All three tables are `unbound` and backed by the [`faker` connector](https://github.com/knaufk/flink-faker).
+
+All sources of tracked visits have the `location` and `visit_time` in commmon. Some have `visitors` some have
+`spacecrafts` and one has both.
+
+```sql
+DROP TABLE IF EXISTS rickandmorty_visits;
```

Comment:
I am more of a fan of self-contained scripts with CREATE TEMPORARY TABLE statements, but we have not aligned on anything yet.

Reply:
`TEMPORARY` seems to be undocumented for `CREATE TABLE`.
https://ci.apache.org/projects/flink/flink-docs-master/docs/dev/table/sql/create/#create-table
i guess i'd leave it as it is.